var/spack/repos/builtin/packages/libgpg-error/package.py

```diff
@@ -32,7 +32,7 @@ class LibgpgError(AutotoolsPackage):
        SmartCard Daemon and possibly more in the future. """
 
     homepage = "https://www.gnupg.org/related_software/libgpg-error"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgpg-error/libgpg-error-1.18.tar.bz2"
```

Comment:
i would keep the old one in `list_url` and add `list_depth` so that it can be a mirror